docs/en/contribution/How-to-release.md

```diff
@@ -7,16 +7,6 @@ This documentation guides the release manager to release the SkyWalking Python i
 1. Close (if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-python](https://github.com/apache/skywalking-python/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
 2. Update CHANGELOG.md and `version` in `pyproject.toml`.
 
-
-## Add your GPG public key to Apache svn
-
-1. Upload your GPG public key to a public GPG site, such as [MIT's site](http://pgp.mit.edu:11371/). 
-
-1. Log in [id.apache.org](https://id.apache.org/) and submit your key fingerprint.
-
-1. Add your GPG public key into [SkyWalking GPG KEYS](https://dist.apache.org/repos/dist/release/skywalking/KEYS) file, **you can do this only if you are a PMC member**.  You can ask a PMC member for help. **DO NOT override the existed `KEYS` file content, only append your key at the end of the file.**
```

Comment:
This step is needed however, the vote and the users need this to check the tars are signed by the release manager.

Reply:
Fixed!